pooch/utils.py

```diff
@@ -40,15 +41,7 @@ def os_cache(project, platform=None):
     ('~', '.cache', 'myproject')
 
     """
-    if platform is None:
-        platform = sys.platform
-    if platform == "darwin":
-        cache_path = Path("~", "Library", "Caches", project)
-    elif platform == "win32":
-        cache_path = Path("~", "AppData", "Local", project, "cache")
-    else:  # *NIX
-        cache_path = Path("~", ".cache", project)
-    return cache_path
+    return appdirs.user_cache_dir(project)
```

Comment:
Should this be `Path(appdirs.user_cache_dir(...))`? The doctest seems to indicate so

Reply:
It should. I changed that before merging.